internal/registry/clientsecretrequest/rest.go

```diff
@@ -70,32 +125,159 @@ func (*REST) Categories() []string {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	t := trace.FromContext(ctx).Nest("create", trace.Field{
-		Key:   "kind",
-		Value: "OIDCClientSecretRequest",
-	})
+	t := trace.FromContext(ctx).Nest("create",
```

Comment:
In unit tests, when we setup a `context` to call `Create()` we can put a trace in the context so we can test the trace  & write assertions against it.